### PR TITLE
TEVA-3730 Change the appearance of the jobseeker survey link component

### PR DESCRIPTION
**Pull request by elliotcm**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-3730

### Before

Screenshot 2022-01-27 at 10 02 47

### After

Screenshot 2022-01-27 at 10 27 53

### CODE REVIEW

File: app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-    color: govuk-colour('black');
-    padding: govuk-spacing(2);
+    .account-survey-cta__content {
+      background-color: govuk-colour('blue');
+      color: govuk-colour('white');
+      padding: govuk-spacing(3) govuk-spacing(4);
 
-    a {
-      color: govuk-colour('black');
+      a.govuk-button {
```
Comment (alexbowen) [inline app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss:20]: should only the class selector here i.e remove the `a.`

File: app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim [hunk 2]
```diff
@@ -1,3 +1,5 @@
 .account-survey-cta
-  .account-survey-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="account-survey-cta"
-    = t(".survey_text_html", link: link_to_survey)
+  .wrapper
+    .account-survey-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="account-survey-cta"
```
Comment (alexbowen) [inline app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim:3]: is the id on this needed? although see you havent added it in this PR

Comment (elliotcm) [reply app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim:3]: Yeah I don't know.

**Review by alexbowen — APPROVED** [review]

File: app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim (hunk 2 above)
Comment (alexbowen) [reply app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim:3]: i guess leave it, its not your change

File: app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss (hunk 1 above)
Comment (elliotcm) [reply app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss]: Done.

**Pull request merged**